src/webhandlers.go

```diff
@@ -208,6 +208,12 @@ func HandleAPIDaysID(w http.ResponseWriter, r *http.Request) {
 	writeJSON(w, js, status)
 }
 
+func HandleAPI404(w http.ResponseWriter, _ *http.Request) {
+	js, err := json.Marshal(map[string]interface{}{"ok": false, "error": "No such API method"})
```

Comment:
Seems like `panicIfError` should be renamed. Logging is it's main goal now.

Reply:
@DarkSquirrelComes, how do you want it to be named?

Reply:
@kolayne This function panics and logs simultaneously. It breaks `single responsibility principle`.

Reply:
@DarkSquirrelComes, well, I can't think of a case when I want to panic because of a non-nil `error`, but don't want this to be logged unlike other problems I panic because of

Reply:
And I also don't understand what kind of fix you're suggesting. I'm just saying there is no better solution coming to my mind (probably because I don't get the problem)